numpy/core/numeric.py

```diff
@@ -707,7 +694,7 @@ def correlate(a, v, mode='valid'):
     --------
     convolve : Discrete, linear convolution of two one-dimensional sequences.
     multiarray.correlate : Old, no conjugate, version of correlate.
-    scipy.signal.correlate : uses FFT which has superior performance on large arrays. 
+    scipy.signal.correlate : uses FFT which has superior performance on large arrays.
```

Comment:
This is tripping up the linter since the line is 85, not 80. But I think it is OK.